products/installation.yaml

```diff
@@ -106,7 +106,7 @@ products:
     channel: alpha
     package: observability-operator
     installFrom: "index"
-    index: "quay.io/rhoas/observability-operator-index:v3.0.10"
+    index: "quay.io/rhoas/observability-operator-index:v3.0.11"
```

Comment:
@valerymo does the installation-cpaas also require a bump?

Reply:
@MStokluska , Thank you very much for this comment. installation-cpaas - updated.